ffmpeg/nvidia_test.go

```diff
@@ -103,6 +103,8 @@ func TestNvidia_Transcoding(t *testing.T) {
 
 func TestNvidia_Pixfmts(t *testing.T) {
 
+	return // This test only seems to work with P100
```

Comment:
is it possible to test for a P100 instead of aborting the whole test?

Reply:
Probably via parsing the output of `nvidia-smi` . I'd have to reconfigure this instance with a P100 to confirm the string to use.

Ideally the test would be made more robust so we wouldn't be gating on specific cards but need to look into our options a little more closely.  Created https://github.com/livepeer/lpms/issues/134 so it isn't lost in the shuffle.